src/liquid/components/ld-tooltip/ld-tooltip.css

```diff
@@ -0,0 +1,225 @@
+@keyframes ld-tooltip-show {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.ld-tooltip {
+  /* TODO: move to global vars? */
```

Comment:
Maybe it's a little too early, because it would result in a file with only one var in it... Or would you also add keyframes for "common animations", like `ld-anim-fade-in`, to that file?

Reply:
Probably I would add easing functions to that file. We can do it later if you think it's too early.